src/plugins/scriptedwizard/resources/wxwidgets/wizard.script

```diff
@@ -1153,18 +1160,100 @@ function OnGetPrevPage_WxAddLib()
 
 function OnEnter_WxAddLib(fwd)
 {
+    if (fwd)
+    {
+        if (IsDLL) // no linking against any winsock needed
+        {
+            Wizard.CheckCheckbox(_T("chkWxLinkWinSock2"), false);
+            Wizard.EnableWindow(_T("chkWxLinkWinSock2"), false);
+        }
+        else
+        {
+            Wizard.CheckCheckbox(_T("chkWxLinkWinSock2"), IntToBool(ConfigManager.Read(_T("/wx_project_wizard/winsock2"), 0)));
+            Wizard.EnableWindow(_T("chkWxLinkWinSock2"), true);
+        }
+    }
     return true;
 }
 
 function OnLeave_WxAddLib(fwd)
 {
     if (fwd)
     {
+        if (GetCompilerFactory().CompilerInheritsFrom(Wizard.GetCompilerID(), _T("gcc*")))
+        {
+            local tempLibArray = ::wxArrayString();
+            
+            tempLibArray = GetArrayFromString(Wizard.GetListboxStringSelections(_T("lstWxLibs")), _T(";"), false);        
+            if (tempLibArray.Index(_T("wxQA")) >= 0)
+            {
+                if (Message(_T("Library wxQA may not be available for wxWidgets built with GCC.\n" +
+                            "Continue anyway?"),
+                            _T("Warning"), wxYES_NO) == wxID_NO)
+                {
+                    return false;
+                }
+            }
+        }
+
+        LibWxLinkWinSock2 = Wizard.IsCheckboxChecked(_T("chkWxLinkWinSock2"));
         AddlLibList = Wizard.GetListboxStringSelections(_T("lstWxLibs"));
     }
     return true;
 }
 
+function OnGetNextPage_WxAddLib()
+{
+    return _T("");
```

Comment:
Add comment why this returns an empty string.

Reply:
Will do.